deployments/liqo/files/liqo-controller-manager-ClusterRole.yaml

```diff
@@ -262,6 +262,25 @@ rules:
   - get
   - patch
   - update
+- apiGroups:
```

Comment:
Wondering why this permissions weren't present before...

Reply:
I think those permissions should be defined elsewhere. Investigating...

Reply:
It's just an issue of misspelled case: resourceOffers, resourceRequest.... Fixing.